### DISCUSSION TITLE
Cross Compiling using Zig

### DISCUSSION THREAD

**Discussion by wwderw** [webui-dev/webui#476] [Q&A] [answered]
I've got a project that I was wanting to build for Windows.  I'm on a Linux machine (I used the current daily as well as .13 "stable" version with the same results).

I am not using a build.zig, but invoking zig build-exe directly using the following command: zig build-exe main.zig -I ../include ../lib/webui.lib --library c -lws2_32 -luser32 -target x86_64-windows-gnu --name main.exe

However, I get this error: error: lld-link: undefined symbol: __declspec(dllimport) CoTaskMemFree
    note: referenced by webui.lib(webui.obj):(TitleChanged_Invoke)

The paths are correct as those same paths do work for the Linux build, but what lib am I missing to link to for that symbol?

**Comment by AlbertShown** [comment, marked as answer]
You need to add `ole32.lib` I guess.

**Reply by wwderw** [reply]
That was it.
